descriptors/ModuleDescriptor-template.json

```diff
@@ -267,6 +299,43 @@
         "proxiesfor.item.put",
         "proxiesfor.item.delete"
       ]
+    },{
+      "permissionName": "user-settings.custom.fields.collection.get",
```

Comment:
Can you rename `custom.fields` to `custom-fields` (similar to `user-settings`)?

Reply:
ok, will be changed